internal/metrics/dashboard/openshift-logging-dashboard.json

```diff
@@ -64,24 +64,14 @@
           },
           "targets": [
             {
-              "expr": "sum by(job, namespace, app_kubernetes_io_name) (increase(vector_component_received_bytes_total{component_kind=\"source\", component_type!=\"internal_metrics\"}[24h]))",
-              "legendFormat": "{{namespace}}/{{job}}/{{app_kubernetes_io_name}}",
+              "expr": "sum by(namespace, app_kubernetes_io_instance) (increase(vector_component_received_bytes_total{component_kind=\"source\", component_type!=\"internal_metrics\"}[24h]))",
```

Comment:
remind me what was the value we added to "..._io_name" ?

Reply:
collector type